app/components/Image/ProfilePicture.tsx

```diff
@@ -1,28 +1,27 @@
-import type { User } from 'app/models';
+import type { UnknownUser } from 'app/store/models/User';
 import CircularPicture from './CircularPicture';
 import type { ComponentProps } from 'react';
 
-type Props = Omit<
-  {
-    user: Pick<
-      User,
-      'profilePicture' | 'username' | 'profilePicturePlaceholder'
-    >;
-    size: number;
-    className?: string;
-  } & ComponentProps<typeof CircularPicture>,
-  'placeholder' | 'src'
->;
+type Props = {
+  user: Pick<
+    UnknownUser,
+    'profilePicture' | 'username' | 'profilePicturePlaceholder'
+  >;
+  size: number;
+  className?: string;
+  alt?: string;
+} & Omit<ComponentProps<typeof CircularPicture>, 'src' | 'alt'>;
 
 const ProfilePicture = ({
   user,
   size = 100,
   className,
+  alt = `${user.username}'s profile picture`,
```

Comment:
Why bother with the alt here when this is the only one that makes sense? Just remove the alt prop alltogether and use this one always

Reply:
Idk. I just thought it was nice to let users override _if_ they wanted to - but yes never should heh